Samples/PublicSamples/RecordingBot/docs/setup/policy.md

```diff
@@ -5,109 +5,121 @@ The following steps should be done by an Office 365 administrator.
+
+### Create the Application Instance
+
+[New-CsOnlineApplicationInstance](https://learn.microsoft.com/en-us/powershell/module/skype/new-csonlineapplicationinstance?view=skype-ps)
+```powershell
+New-CsOnlineApplicationInstance
+   -UserPrincipalName <the email of the generated micrsoft entra id resource> `
+   -DisplayName <a name for the generated microsoft entra id resource> `
+   -ApplicationId <the Application Id of the Bot Application Registration>
+```
+The Application Id might not be from your own Microsoft Entra ID,
+but from the Microsoft Entra ID that hosts this Bot.
+So basically from the Service Provider.
+
+You will now have to Synchronize this Application Instance into the Agent Provisioning Service
+```powershell
+Sync-CsOnlineApplicationInstance
+    -ObjectId <the object id of the Application Instance>
+    -ApplicationId <the application id of the remote application registration>
+```
+You can get the Object Id by executing the `Get-CsOnlineApplicationInstance -Displayname <the name you provided>` and checking the output.
+The Object Id will also be displayed after the `New-CsOnlineApplicationInstance` command.
+The Application Id is the same Application Id you used to create the Application Instance.
+
+### Create the policy
+
+[New-CsTeamsComplianceRecordingPolicy](https://learn.microsoft.com/en-us/powershell/module/skype/new-csteamscompliancerecordingpolicy?view=skype-ps)
+```powershell
+New-CsTeamsComplianceRecordingPolicy
+   -Identity <provide a name for your policy>
+   -Enabled $true
+```
+With this, you will just have a policy.
+You can already assign this policy to users, but it will not do anything,
+because it does not have any Recording Applications assigned to it yet.
+
+### Create the Recording Application
+
+[New-CsTeamsComplianceRecordingApplication](https://learn.microsoft.com/en-us/powershell/module/skype/new-csteamscompliancerecordingapplication?view=skype-ps)
+```powershell
+New-CsTeamsComplianceRecordingApplication
+   -Parent <Recording Policy Name>
+   -Id <Application Instance Object Id>
+```
+The <Recording Policy Name> is the Parameter `Identity` from the `New-CsTeamsComlianceRecordingPolicy`.
+So the Parent of a Recording Application is a Recording policy.
+The <Application Instance Object Id> is the Object ID of the Object created with `New-CsOnlineApplicationInstance`.
+So basically you now have
+- assigned an Application in your own Entra (Application Instance)
+ - wich points to an application on an external Entra (Application Id that was assigned to Application Instance)
+- to a Teams Compliance Policy (by using its name)
+
+### Activate the policy
+
+[Grant-CsTeamsComplianceRecordingPolicy](https://learn.microsoft.com/en-us/powershell/module/skype/grant-csteamscompliancerecordingpolicy?view=skype-ps)
+```powershell
+Grant-CsTeamsComplianceRecordingPolicy
+     -Identity <User Principal name of the Application Instance>
```

Comment:
Shouldn't the Identity be the user principal name of the user we want to have under Recording Policy not the principal name of the Application Instance?

Reply:
No it should not

Reply:
```
-Identity
Indicates the Identity of the user account to be assigned the per-user Teams recording policy. User Identities can be specified using one of the following formats:

the user's SIP address;
the user's user principal name (UPN);
the user's Active Directory display name (for example, Ken Myer).
```

sure? This is what the docu you linked states about the Identity Parameter.

Reply:
Yes i am sure, the user used here is the user that was created with the Application Instance
the rest of the grants are based on the administrators consent

Reply:
To be honest this doesn't make sense to me because it is not necessary to _activate_ the policy or does bring any benefit.

Reply:
it is how it is
i cant change how the module works
nor how teams administration works

Reply:
But it is not how the Module works xD, the Policy I created for our Test Tenant didn't need this and also works fine

Reply:
If you know this better
be my guest
this is what the documentation did before
i just did this process once
and i am not a teams administrator
so... if you need to remove this, then do so

Reply:
I think there is the misunderstanding in the previous documentation about this line, it means we need the policy to be created with a Application Instance assigned to Grant the policy to a user not that we need to assign the policy to the upn of the ApplicationInstance